lib/libraries/c3/extensions/default-date-format.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function(a, b){
+  var d = Math.abs(new Date(a).getTime() - new Date(b).getTime());
```

Comment:
Sorry to be nitpicky, but is there a better variable name to describe what `d` is?